internal/js/modules/k6/browser/tests/get_by_test.go

```diff
@@ -1296,3 +1296,53 @@ func TestGetByTextSuccess(t *testing.T) {
 		})
 	}
 }
+
+func TestGetByNullHandling(t *testing.T) {
+	t.Parallel()
+
+	tb := newTestBrowser(t, withFileServer())
+	tb.vu.ActivateVU()
+	tb.vu.StartIteration(t)
+
+	// Setup
+	tb.vu.SetVar(t, "page", &sobek.Object{})
+	_, err := tb.vu.RunAsync(t, `
+		page = await browser.newPage();
+	`)
+	require.NoError(t, err)
+
+	_, err = tb.vu.RunAsync(t, `
+		await page.getByRole().click();
+	`)
+	require.ErrorContains(t, err, "missing required argument 'role'")
```

Comment:
There is a lot of logic in the mapping layer that should be tested, not just mocked. This is why I have used this approach to testing rather than pure Go tests. Let me know what you think.

Reply:
I believe these tests are valuable, but they are not type-safe and becomes very difficult to adopt later on. For example, we can't easily find which tests use which Go API (unless we do text search rather than "find callers" in our IDEs). I currently donmt have a nice idea for a solution. I usually write Go tests if the mapping logic is simple. Or separate the mapping logic in a other function and write a test for it.